Matrix/src/com/dwardu/Matrix/MatrixFunction.java

```diff
@@ -335,11 +335,25 @@ private static double getTwoByTwoDeterminant(Matrix a) {
 				* a.getValueAt(1, 0);
 	}
 	/*
-	 * Returns the transpose of a matrix
+	 * Returns the transpose of a matrix 
+	 *
+     * @param matrix is the matrix to be transposed
+     *
+	 * @return transpose of matrix
 	 */
 	public static Matrix transpose(Matrix matrix)
 	{
-		return null;
+
+		Matrix transposed = new Matrix(matrix.getColumnSize(), matrix.getRowSize());
+		
+		for (int row = 0; row < matrix.getRowSize()){

```

Comment:
Can you remove the spacing between the lines in the for loops to make them the same as the other methods please?